config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
```

Comment:
The BCrypt hashes have always the same length, therefore they don't need to vary in length

Reply:
Have you considered making that a constraint? I'm not sure if `character` handles empty as null, I don't think it does, so empty character or lengths up to 59 would violate that assumption.

Reply:
Good Point. It just learned that the char datatype doesn't enforce a specific length, but just fills the unused chars (probably with NULLS),